### PR TITLE
Change CLI init config branding from Aptos to Movement

### PR DESCRIPTION
**Pull request by andygolay**
## Description

A couple simple cosmetic branding fixes for the CLI, and update output directory.
* Change "Aptos already initialized" to "Movement already initialized" 
* Change `aptos init` to `movement init`
* Update output directory to `.movement`

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ x ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ x ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?
Just a couple simple cosmetic fixes; visually evaluated the code changes.

## Key Areas to Review
Ensure there was no careless typo in the changes.

## Checklist
- [ x ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ x ] I have made corresponding changes to the documentation

### CODE REVIEW

File: crates/aptos/src/common/init.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@ impl CliCommand<()> for InitTool {
 
         // Select profile we're using
         let mut profile_config = if let Some(profile_config) = config.remove_profile(profile_name) {
-            prompt_yes_with_override(&format!("Aptos already initialized for profile {}, do you want to overwrite the existing config?", profile_name), self.prompt_options)?;
+            prompt_yes_with_override(&format!("Movement already initialized for profile {}, do you want to overwrite the existing config?", profile_name), self.prompt_options)?;
```
Comment (l-monninger) [inline crates/aptos/src/common/init.rs:104]: You have updated the directory output to be `.movement`, right?

Comment (andygolay) [reply crates/aptos/src/common/init.rs:104]: Yes, I just added [a commit](https://github.com/movementlabsxyz/aptos-core/pull/49/commits/87b7ef250c181a8937c472f08af5c962cd7fa2eb) to update it.

Comment (l-monninger) [reply crates/aptos/src/common/init.rs:104]: Okay, let me test this e2e and then hopefully I will be able to approve.

**Review by l-monninger — APPROVED** [review]
moto bene

**Pull request merged**